tool/pom.xml

```diff
@@ -6,7 +6,10 @@
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<parent>
+    <properties>
```

Comment:
Please use the same indentation style within one file (tabs here). If you use IDEA or another IDE you can turn on highlighting of leading spaces.